detection-rules/link_multifactor_authentication_QR_code_phishing.yml

```diff
@@ -0,0 +1,62 @@
+name: "Link: Multi-Factor Authentication phishing via QR code"
+description: |
+  This rule detects the verbiage related to Multi-factor authentication in the subject, body or attachments. The rule also checks for the presence of a QR code with a URL
+  that contains the recipients email address, from an unsolicited sender.
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound 
+  
+  // body html, plain.raw subject or any attachments contain QR|2fa|MFA|Multi factor authentiation
+  
+  and 1 of (
+      (
+          regex.icontains(subject.subject, '(\b2fa\b|\bQ.?R\.?\s?\b|MFA|Muti[ -]?Factor Auth(entication)?)')
+      ),
+      (
+          regex.icontains(body.html.display_text, '(\b2fa\b|\bQ.?R\.?\s?\b|MFA|Muti[ -]?Factor Auth(entication)?)')
+      ),
+      (
+          regex.icontains(body.plain.raw, '(\b2fa\b|\bQ.?R\.?\s?\b|MFA|Muti[ -]?Factor Auth(entication)?)')
+      ),
+      (
```

Comment:
idk if this `(` is standard but it seems pretty verbose to me. and `1 of` is technically `or` (gets turned into it anyway)